tests/test_data_client.py

```diff
@@ -26,16 +30,20 @@
 LOCATION_IDS = [LOCATION_ID]
 ORG_ID = "organization_id"
 ORG_IDS = [ORG_ID]
-MIME_TYPES = ["mime_type"]
-START_DATETIME = datetime(2001, 1, 1, 1, 1, 1)
-END_DATETIME = datetime(2001, 1, 1, 1, 1, 1)
+MIME_TYPE = "mime_type"
+MIME_TYPES = [MIME_TYPE]
+URI = "some.robot.uri"
 SECONDS_START = 978310861
 NANOS_START = 0
 SECONDS_END = 978310861
 NANOS_END = 0
+START_TS = Timestamp(seconds=SECONDS_START, nanos=NANOS_START)
+END_TS = Timestamp(seconds=SECONDS_END, nanos=NANOS_END)
+START_DATETIME = START_TS.ToDatetime()
+END_DATETIME = END_TS.ToDatetime()
 TAGS = ["tag"]
 BBOX_LABELS = ["bbox_label"]
-FILTER = Filter(
+FILTER = DataClient.create_filter(
```

Comment:
(flyby) we both had a hardcoded filter here and - in one method below - a filter created via `create_filter`. this seemed redundant so I moved to a default of only using a single filter that is created via our helper method.